docker-compose.dev.yml

```diff
@@ -14,7 +14,7 @@ services:
       - core_cms_net
 
   postgres:
-    image: postgres:11.5
+    image: postgres:12
```

Comment:
Should this require any changes to the database beyond `migrations`? (I think and hope not, just asking if you noticed anything about it.)

Reply:
From this branch, on my existing local Core-CMS, I ran `make build` and `make start` and I got:

```
2023-08-28 16:54:50 
2023-08-28 16:54:50 PostgreSQL Database directory appears to contain a database; Skipping initialization
2023-08-28 16:54:50 
2023-08-28 16:54:50 2023-08-28 21:54:50.866 UTC [1] FATAL:  database files are incompatible with server
2023-08-28 16:54:50 2023-08-28 21:54:50.866 UTC [1] DETAIL:  The data directory was initialized by PostgreSQL version 11, which is not compatible with this version 12.16 (Debian 12.16-1.pgdg120+1).
```

This crashed the database container. I fear this would affect existing CMS on a VM. Is that fear warranted?

_Reverting to `main` and re-`build`ing and re-`start`ing **did** fix the database container._

Reply:
**Update**: I learned this is not supposed to work on local yet. There is an Infrastructure task to migrate the databases to v12.